server_integration_test.sh

```diff
@@ -21,11 +21,11 @@ if [ $output -eq "200" ];
 		echo "FAILED: TO RECEIVE A RESPONSE"
 fi
 
-# === Integration Test 2 ===
-# Checking that the server can return a static file
-output2=$(curl http://localhost:1890/static/test.html | diff example_files/test.html - )
+# # === Integration Test 2 ===
+# # Checking that the server can return a static file
+output2=$(curl http://localhost:2020/test.html | diff example_files/test.html - )
```

Comment:
I actually changed the StaticHandler's url prefix back to `/static` in `test_config` file in order to test `NotFoundHandler`